examples/archive/bettinggame/bettinggame.cabal

```diff
@@ -0,0 +1,54 @@
+cabal-version: 3.4
+
+name:           bettinggame
```

Comment:
What do you think of making one `archive.cabal` package and put the contracts we no longer want to maintain there? The betting game is just one, but @lucaspena's onboarding contract would be a great one to rescue from git too.

Reply:
I considered that first, but it has the disadvantage that if one is only interested in building a particular contract, one has to edit the cabal configuration to disable building the dependencies and the local modules that are only relevant for the other contracts, and one also needs to select the tests to run on the command line.

Reply:
But I thought the archive was meant for contracts we don't want to build anymore

Reply:
That is still the case. The discussion here is whether to have a single cabal package in the archive containing all of the contracts, or to have a separate cabal package for each contract in the archive.